doc/Programs/03-environment-variables.pod6

```diff
@@ -154,15 +217,45 @@ X<|RAKUDO_SNAPPER>
 Indicates the period in which the telemetry snapper will take a snapshot.
 Defaults to .1 for 10 snapshots per second.
 
-=head1 AUTHORS
+X<|RAKUDO_HOME>
+=item C<RAKUDO_HOME>
+
+Allows to override the Raku installation path. Defaults to
+C<[rakudo_executable_dir]/../share/perl6> in relocatable builds and the
+absolute path to that folder in non-relocatable builds.
+
+X<|NQP_HOME>
+=item C<NQP_HOME>
+
+Allows to override the NQP installation path. Defaults to
+C<[rakudo_executable_dir]/../share/nqp> in relocatable builds and the absolute
+path to that folder in non-relocatable builds.
+
+=head1 WINDOWS PECULIARITIES
+
+X<|rakuw.exe>X<|rakudow.exe>
+=head2 Non-console applications
 
-Initial version written by the Rakudo contributors, see the
-L<CREDITS file|https://github.com/rakudo/rakudo/blob/master/CREDITS>.
+On Windows programs are compiled to either be I<console> applications or
+I<non-console> applications. I<Console> applications always open a console
+window. There is no straightforward way to suppress this window.
+
+Rakudo provides a separate set of executables suffixed with a C<'w'>
+(C<rakuw.exe>, C<rakudow.exe>, ...) that are compiled as I<non-console>
+applications. These do not spawn this console window.
+
+B<WARNING> By default these I<non-console> applications will silently swallow
+everything that is printed to C<STDOUT> and C<STDERR>.
+
+To receive the output of the program it suffices to redirect it externally:
+
+  rakuw.exe script.raku >stdout.txt 2>stderr.txt
+
+=head1 AUTHORS
 
-The L<initial version of this manual
-page|https://github.com/rakudo/rakudo/blob/master/docs/running.pod> was written
-by Reini Urban, Moritz Lenz and the Rakudo contributors.
+Written by the Rakudo contributors, see the CREDITS file.
```

Comment:
Why not keep the original links?

Reply:
If we manage to keep the files in sync, there is no reason to link there. Also the link makes no sense in the rakudo/rakudo version of the file and I'd like to keep the differences we need to maintain minimal.